templates/brand/resources.html

```diff
@@ -0,0 +1,1010 @@
+                  <i class="p-icon--sort-both"
+                     style="width: 18px;
+                            height: 18px;
+                            margin-bottom: 0.5rem"></i>
+                </div>
+
+                <div class="icon__item">
+                  <i class="p-icon--sort-down"
+                     style="width: 18px;
+                            height: 18px;
+                            margin-bottom: 0.5rem"></i>
+                </div>
+
+                <div class="icon__item">
+                  <i class="p-icon--sort-up"
+                     style="width: 18px;
+                            height: 18px;
+                            margin-bottom: 0.5rem"></i>
+                </div>
+
+                <div class="icon__item">
+                  <i class="p-icon--starred"
+                     style="width: 18px;
+                            height: 18px;
+                            margin-bottom: 0.5rem"></i>
+                </div>
+
+                <div class="icon__item">
+                  <i class="p-icon--status-failed-small"
+                     style="width: 18px;
+                            height: 18px;
+                            margin-bottom: 0.5rem"></i>
+                </div>
+
+                <div class="icon__item">
+                  <i class="p-icon--error"
+                     style="width: 18px;
+                            height: 18px;
+                            margin-bottom: 0.5rem"></i>
+                </div>
+
+                <div class="icon__item">
+                  <i class="p-icon--status-in-progress-small"
+                     style="width: 18px;
+                            height: 18px;
+                            margin-bottom: 0.5rem"></i>
+                </div>
+
+                <div class="icon__item">
+                  <i class="p-icon--status-in-progress"
+                     style="width: 18px;
+                            height: 18px;
+                            margin-bottom: 0.5rem"></i>
+                </div>
+
+                <div class="icon__item">
+                  <i class="p-icon--status-queued-small"
+                     style="width: 18px;
+                            height: 18px;
+                            margin-bottom: 0.5rem"></i>
+                </div>
+
+                <div class="icon__item">
+                  <i class="p-icon--status-queued"
+                     style="width: 18px;
+                            height: 18px;
+                            margin-bottom: 0.5rem"></i>
+                </div>
+
+              </div>
+            </div>
+
+            <label class="p-switch">
+              <input type="checkbox"
+                     class="p-switch__input"
+                     role="switch"
+                     id="icon_switch"
+                     onchange="switchIconThumbTheme()" />
+              <span class="p-switch__slider"></span>
+              <span class="p-switch__label">Dark mode</span>
```

Comment:
Is the dark mode toggle yet to be updated? It changes the icons but because the background is still white, the icons don't show up as expected

Reply:
Yes, this should be addressed when refactoring code for all the pages.